plugins/polio/js/src/forms/BudgetForm.js

```diff
@@ -47,26 +66,164 @@ export const BudgetForm = () => {
         });
         return totalPopulation ? (totalCost / totalPopulation).toFixed(2) : '-';
     }, [rounds]);
+    const hasRequestFieldsError = findErrorInFieldList(
+        BUDGET_REQUEST,
+        errors,
+        touched,
+    );
+    const hasRRTReviewError = findErrorInFieldList(RRT_REVIEW, errors, touched);
+    const hasORPGReviewError = findErrorInFieldList(
+        ORPG_REVIEW,
+        errors,
+        touched,
+    );
+    const hasApprovalFieldsError = findErrorInFieldList(
+        REVIEW_FOR_APPROVAL,
+        errors,
+        touched,
+    );
 
     return (
         <>
             <Grid container spacing={2}>
                 <Grid container direction="row" item spacing={2}>
                     <Grid xs={12} md={6} item>
+                        <Box mb={2}>
+                            <Typography variant="button">
+                                {formatMessage(MESSAGES.budgetApproval)}
+                            </Typography>
+                        </Box>
+                        <Box mb={2}>
+                            <Divider style={{ height: '1px', width: '100%' }} />
```

Comment:
is this style required ?

Reply:
I tried without, the `Divider`was invisible